docs/docusaurus/docs/application_integration_support.md

```diff
@@ -22,7 +22,7 @@ The following table defines the GX Cloud, GX Core, and Community Supported integ
 
 | Integration Type                 | GX Cloud                              | GX Core                                                                                    | Community            |
 | -------------------------------- | ------------------------------------- | ------------------------------------------------------------------------------------------ | -------------------- |
-| Data Sources<sup>1</sup>         | Snowflake<br/> PostgreSQL<sup>2</sup> | Snowflake<br/>PostgreSQL<br/>Sqlite<br/>Databricks (SQL)<br/>BigQuery<br/>Spark<br/>Pandas | MSSQL<br/>MySQL<br/> |
+| Data Sources<sup>1</sup>         | Snowflake<br/>Databricks (SQL)<br/> PostgreSQL<sup>2</sup> | Snowflake<br/>PostgreSQL<br/>Sqlite<br/>BigQuery<br/>Spark<br/>Pandas | MSSQL<br/>MySQL<br/> |
```

Comment:
Is Databricks SQL supported in GX Core as well? If so, let's add it to the table too.

Reply:
My mistake. Good catch.